src/components/configure/AesConfigure.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Form, Input, Button } from 'antd';
+
+const AesConfigure = ({ form, aes, handleSubmit }) => {
+  const { getFieldDecorator } = form;
+
+  const handleSubmitClick = e => {
+    e.preventDefault();
+    form.validateFields((err, values) => {
+      if (!err) {
+        handleSubmit(values.aes);
+      }
+    });
+  }
+  return (
+    <div>
+      <p>This key is used by the authorization module in Space Cloud to encrypt security rules</p>
```

Comment:
Replace this with `This key is used by the security rules in Space Cloud to encrypt/decrypt certain fields`